### PR TITLE
task_01.py

### PR DESCRIPTION
**Pull request by Lalanikw**

### CODE REVIEW

File: task_01.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        fc (decimal): value
+
+    returns:
+        temperature in celcius
+
+    Example:
+        >>>fahrenhiet_to_celsius(30.0)
+        -1.111
+    """
+    fc = fahrenhiet_to_celsius(degrees) = (degrees - 32) * 5) / 9
```
Comment (cheuschober) [inline task_01.py]: @Lalanikw You ended up re-declaring your function here which would have been an illegal task, in this case. Additionally, you needed to process all input as decimals in order to return decimals per the directions.

File: task_01.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    returns:
+        temperature in kelvin
+
+    Example:
+        >>>celsius_to_kelvin(10)
+        283.15
+    """
+    
+    ck = degrees + ABSOLUTE_DIFFERENCE
```
Comment (cheuschober) [inline task_01.py]: Doesn't return decimal.

File: task_01.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    Examples:
+        >>> fahrenhiet_to_kelvin(30.2)
+        272.26
+    """
+    
+    fc = fahrenhiet_to_celsius(degrees)
+    fk = celsius_to_kelvin(fc)
+    return fk
+    fk = fc + ABSOLUTE_DIFFERENCE
+    return fk
```
Comment (cheuschober) [inline task_01.py]: @Lalanikw Too many returns here.

Comment (Lalanikw) [inline task_01.py]: Hi Professor,

Looks like I have 0 for this assignment? Can I try again?

Also, I have very low points for all assignments. Can I re-submit them?

Thanks,
Lalani.

Comment (cheuschober) [inline task_01.py]: @Lalanikw I answered you by e-mail.

**Pull request merged**